Cubical/HITs/Bouquet/FreeGroupoid/Base.agda

```diff
@@ -0,0 +1,29 @@
+{-
+
+This file contains:
+
+- An implementation of the free groupoid (a free group that has no limitiations
+  over the high dimensional path structure). An intermediate construction used
+  to calculate the fundamental group of a Bouquet.
+
+-}
+{-# OPTIONS --safe #-}
+
+module Cubical.HITs.Bouquet.FreeGroupoid.Base where
```

Comment:
Why is this part of the `Bouquet` folder? I think it should be its own folder in `HITs`

Reply:
At first it seemed to me a little bit too specific but I think you are right  and now I'm moving it to its own folder.